problems/objects.js

```diff
@@ -151,15 +191,29 @@ const authorScores = () => {};
  * @returns {string} The full name of person with best score.
  */
 
-const bestScore = () => {};
+const bestScore = (submissions) => {
+  let highestScore = {score: -1};
```

Comment:
My only comment here is that if the highest score was less than -1 this would not work.

When doing smallest and largest numbers we want to Infinity/-Infinity